HW4/array_manipulations.cpp

```diff
@@ -0,0 +1,94 @@
+#include "array_manupulations.h"
+#include <chrono>
+#include <iomanip>
+#include <iostream>
+#include <random>
+
+void output_array_range(unsigned int *begin, unsigned int *end) {
+  if (begin == nullptr || end == nullptr)
+    return;
+  for (auto it = begin; it != end; ++it)
+    std::cout << *it << " ";
+  std::cout << std::endl;
+}
+void output_array_range(int *begin, int *end) {
+  if (begin == nullptr || end == nullptr)
+    return;
+  for (auto it = begin; it != end; ++it)
+    std::cout << *it << " ";
+  std::cout << std::endl;
+}
+void output_array_range(double *begin, double *end) {
+  if (begin == nullptr || end == nullptr)
+    return;
+
+  for (auto it = begin; it != end; ++it)
+    std::cout << std::fixed << std::setprecision(2) << *it << " ";
+  std::cout << std::endl;
+}
+void output_array_range(char *begin, char *end) {
+  if (begin == nullptr || end == nullptr)
+    return;
+  for (auto it = begin; it != end; ++it)
+    std::cout << *it << " ";
+  std::cout << std::endl;
+}
+
+void fill_array(int *begin, int *end) {
+  if (begin == nullptr || end == nullptr)
+    return;
+  int kMinRandomRange = 0;
+  int kMaxRandomRange = 10;
```

Comment:
```suggestion
  const int kMinRandomRange {0};
  const int kMaxRandomRange {10};
```
Замінити копіюючу ініціалізацію універсальною.
Якщо в коді змінна не змінюється то це константа